ocaml/lambda/translcore.ml

```diff
@@ -1624,34 +1654,34 @@ and transl_match ~scopes ~arg_sort ~return_sort e arg pat_expr_list partial =
       assert (static_handlers = []);
       let mode = transl_alloc_mode alloc_mode in
       let argl = List.map (fun a -> (a, Sort.for_tuple_element)) argl in
-      Matching.for_multiple_match ~scopes ~return_layout e.exp_loc
+      Matching.for_multiple_match ~scopes ~return_layout loc
         (transl_list_with_layout ~scopes argl) mode val_cases partial
     | {exp_desc = Texp_tuple (argl, alloc_mode)}, _ :: _ ->
         let argl = List.map (fun a -> (a, Sort.for_tuple_element)) argl in
         let val_ids, lvars =
           List.map
             (fun (arg,s) ->
-               let layout = layout_exp s arg in
-               let id = Typecore.name_pattern "val" [] in
-               (id, layout), (Lvar id, s, layout))
+                let layout = layout_exp s arg in
```

Comment:
There are a few places in this diff where the indentation level is modified for code blocks unrelated to the PR. (Editors misbehave sometimes.) Would you mind going through and undoing these pieces of the diff? Doing this work reduce the chance (and impact) of merge conflicts with upstream.

Reply:
Removed this whitespace with a later commit